pkg/controller/volume/attachdetach/cache/actual_state_of_world.go

```diff
@@ -67,11 +67,9 @@ type ActualStateOfWorld interface {
 	SetVolumeMountedByNode(volumeName api.UniqueVolumeName, nodeName string, mounted bool) error
 
 	// ResetNodeStatusUpdateNeeded resets statusUpdateNeeded for the specified
-	// node to false indicating the AttachedVolume field of the Node's Status
-	// object has been updated.
-	// If no node with the name nodeName exists in list of attached nodes for
-	// the specified volume, an error is returned.
-	ResetNodeStatusUpdateNeeded(nodeName string) error
+	// node to true indicating the AttachedVolume field of the Node's Status
+	// object needs to be updated by the node updater again.
+	ResetNodeStatusUpdateNeeded(nodeName string)
```

Comment:
Revert the `If no node with the...` portion of the comment since it is still applicable.